homeassistant/components/vicare/sensor.py

```diff
@@ -220,6 +220,14 @@ class ViCareSensorEntityDescription(SensorEntityDescription, ViCareRequiredKeysM
         unit_getter=lambda api: api.getGasSummaryConsumptionHeatingUnit(),
         state_class=SensorStateClass.TOTAL_INCREASING,
     ),
+    ViCareSensorEntityDescription(
+        key="gas_summary_consumption_heating_lastsevendays",
+        name="Heating gas consumption last seven days",
+        native_unit_of_measurement=UnitOfVolume.CUBIC_METERS,
+        value_getter=lambda api: api.getGasSummaryConsumptionHeatingLastSevenDays(),
+        unit_getter=lambda api: api.getGasSummaryConsumptionHeatingUnit(),
+        state_class=SensorStateClass.TOTAL_INCREASING,
+    ),
```

Comment:
Please place this under the `gas_summary_consumption_heating_currentyear` one, since this sensor is a `lastsevendays` one, and it's in between of the `current` ones. This is to avoid confusion.

Reply:
Thanx for review. Will do.